Tests/Integration/PauseWithPredisTest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+namespace Resque\Plugins\Tests\Integration;
+
+use Resque;
+use Predis\Client;
+
+/**
+ * Pause tests.
+ *
+ * @package     PHP Resque Pause
+ * @author      Wedy Chainy <wedy.chainy@bigcommerce.com>
```

Comment:
Looks like a new file, so I'd modify the author. I'm happy to have tools here instead of Gabe if you like.